src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import './Todo.css'
+import TodoList from './TodoList'
+import TodoForm from './TodoForm'
+
+
+
+const newTodo = [
+    {
+      task: 'Organize Garage',
+      id: 1528817077286,
+      completed: false
+    },
+    {
+      task: 'Bake Cookies',
+      id: 1528817084358,
+      completed: false
+    }
+  ];
+
+class Todo extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+
+            moreTodo: newTodo,
+            task: '',
+            id: Date.now(),
+            completed: '',
+            
+        };
+    }
+
+
+    handleChanges = event => {
+        this.setState({
+            [event.target.name]: event.target.value
+        });
+    };
+
+    updateTodo = event => {
+        event.preventDefault();
+        const myList = {
+            task: this.state.task,
+            id: this.state.id,
```

Comment:
Instead of taking the `id` from state, you can just assign `id: Date.now()` and get rid of `state.id` all together.